src/main/java/org/sagebionetworks/bridge/reporter/helper/BridgeHelper.java

```diff
@@ -98,6 +117,59 @@ public final void setBridgeClientManager(ClientManager bridgeClientManager) {
         return retList;
     }
     
+    public Map<StudyParticipant, RequestInfo> getRequestInfoForParticipant (String studyId, List<StudyParticipant> participants)
+            throws IOException {
+    	Map<StudyParticipant, RequestInfo> retMap = new HashMap<>();
+        
+        ForWorkersApi workersApi = bridgeClientManager.getClient(ForWorkersApi.class);
+        for (StudyParticipant participant : participants) {
+        	RequestInfo requestInfo = workersApi.getRequestInfoForWorker(studyId, participant.getId()).execute().body();
+    		retMap.put(participant, requestInfo);
+    		doSleep();
+        }
+        return retMap;
+    }
+    
+    public Map<RequestInfo, ActivityEventList> getActivityEventForParticipant (String studyId, List<StudyParticipant> participants)
```

Comment:
I would key this map by something like user ID. I'm not 100% sure that two RequestInfo objects generated by the Swagger codegen libraries won't have the same hashCode (value equality). User ID is guaranteed to be unique per user.